.circleci/config.yml

```diff
@@ -113,7 +113,7 @@ mysql-container: &mysql-container
     MYSQL_DATABASE: circleci
 
 postgres-container: &postgres-container
-  image: cimg/postgres:10.20
+  image: cimg/postgres:13.15
   environment:
     POSTGRES_USER: postgres
     POSTGRES_DB: systemdb
```

Comment:
```suggestion
```

Reading here, I see some errors. Also maybe DB was created because of the env variable we set.

https://app.circleci.com/pipelines/github/3scale/porta/28358/workflows/e5d7d425-5adf-47c1-a3c5-c4787edd06de/jobs/317680/parallel-runs/0/steps/0-1

Looking at image docs https://circleci.com/developer/images/image/cimg/postgres it might make sense to just run the iage without custom env variables and see if it will behave better.

Reply:
Great tip @akostadinov !
It seems to have worked :raised_hands: